source/ScreensaverHero.hx

```diff
@@ -0,0 +1,36 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+using flixel.util.FlxSpriteUtil;
+
+// Part 2: Added a Screensaver Hero Class
+class ScreensaverHero extends FlxSprite
+{
+	// Implement Constructor
+	public function new()
+	{
+		// Invoke parent class constructor
+		super(x, y);
+		this.x = x;
+		this.y = y;
+		// Make a simple rectangle sprite
+		var myRectangle = new FlxSprite(x, y);
+		makeGraphic(30, 10, true);
+		// Randomize the direction for the sprite to move each frame
+		var x_m = FlxG.random.int(5, 50);
+		var y_m = FlxG.random.int(5, 50);
+		// Make the graphic move automatically each frame
+		velocity.set(x_m, y_m);
+	}
+
+	// Override inherited update() function/method
+	override public function update(elapsed:Float)
+	{
+		// Set the FlxSprite color property to CYAN and make it move each frame
+		color = 0xFF00FFFF; // FlxColor.CYAN
```

Comment:
The goal of the assignment is to make it possible for the color to change as the ScreensaverHero moves across the screen:

> Inside the ScreensaverHero's update() function, set its [color property](https://api.haxeflixel.com/flixel/FlxSprite.html#color) to a value that is somehow determined by screen position (how is up to you).

You are setting the color property inside of the update function, so we're halfway there. Now you need to make the value you're putting into `color` be determined by your screen position in some way.

It could be as simple as "if it's on the left side of the screen, it's blue, if it's on the right side it's green" or you could use some method like `FlxColor.fromRGB()` to generate a color, using the `x` and `y` properties of your `ScreensaverHero` to create numbers you pass into the method.

We'll need to get this done before we can merge.